spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/thymeleaf/ThymeleafProperties.java

```diff
@@ -38,6 +39,8 @@
 
 	public static final String DEFAULT_SUFFIX = ".html";
 
+	public static final String DEFAULT_ROOT_VIEW = "index";
```

Comment:
We already use the term "welcome page" for when there's an `index.html` file available in a static resource location. For consistency, I wonder if we'd be better reusing that term rather than introducing "root view".